homeassistant/components/sleepiq/select.py

```diff
@@ -114,3 +122,46 @@ async def async_select_option(self, option: str) -> None:
         self._attr_current_option = option
         await self.coordinator.async_request_refresh()
         self.async_write_ha_state()
+
+
+class SleepIQCoreClimateTempSelectEntity(
+    SleepIQSleeperEntity[SleepIQDataUpdateCoordinator], SelectEntity
+):
```

Comment:
What's the name of this entity, will it be `f"SleepNumber {bed.name} Foundation Preset"` as set in the base entity or something else?

Reply:
f"SleepNumber {bed.name} {sleeper.name} Core Climate" 

Which is correct, and tested. Which came out with my bed name and my name for my bed.